Application/Source/Detail/View/DetailViewControllerStyle.swift

```diff
@@ -6,24 +6,27 @@ struct DetailViewControllerStyle: Style {
     let theme: Theme
     let background: BackgroundViewStyle
     let title: LabelStyle
-    let selectionResult: LabelStyle
+    let alternateCopy: AlternateLabelStyle
     let button: ButtonStyle
 
     init(theme: Theme) {
         self.theme = theme
         background = BackgroundViewStyle(theme: theme)
         title = LabelStyle(theme: theme)
-        selectionResult = LabelStyle(theme: theme)
+        alternateCopy = AlternateLabelStyle(theme: theme)
         button = ButtonStyle(theme: theme)
     }
 
     func apply(to styleable: DetailViewController) {
         let view = styleable.detailView
-
         background.apply(to: view)
+
         title.apply(to: view.title)
         title.apply(to: view.contentsListTitle)
-        selectionResult.apply(to: view.selectionResult)
```

Comment:
Why was this removed?

Reply:
Using `alternateCopy` for both `view.selectionResult` and `view.contentsList`. 

Or would you have a separate property in this struct for each UI element, even if they share a Style? That seems like it wouldn't be very scalable for complex views?